databricks-sdk-java/src/main/java/com/databricks/sdk/core/http/Request.java

```diff
@@ -16,6 +16,7 @@ public class Request {
   public static final String PATCH = "PATCH";
   private final String method;
   private String url;
+  private boolean redirectDisabled;
```

Comment:
minor comment: We can also take note of if client has explicitly provided this value.

private Optional<Boolean> redirectBehavior = Optional.empty();

setRedirectBehavior(boolean redirectBehaviorValue) {
  this.redirectBehavior = Optional.of(redirectBehaviorValue);
}

Optional<Boolean> getRedirectBehavior() {
  return redirectBehavior;
}

in CommonsHttpClient:

if (!in.getRedirectBehavior().isEmpty()) {
  request.getParams().setParameter("http.protocol.handle-redirects", in.getRedirectBehavior().get());
}

Reply:
Sure, fixed.